.github/workflows/swift_package_test.yml

```diff
@@ -99,6 +111,18 @@ on:
         type: string
         description: "Command to use when building the package with the Swift SDK for Wasm"
         default: "swift build"
+      android_sdk_build_command:
+        type: string
+        description: "Command to use when building the package with the Swift SDK for Android"
+        default: "swift build"
+      android_sdk_triple:
```

Comment:
I think this property would be particularly useful to express as an array

Reply:
Done